gems/smithy/spec/fixtures/service_index/model.smithy

```diff
@@ -4,7 +4,7 @@ namespace smithy.ruby.tests
 
 service ServiceIndex {
     version: "1.0.0"
-    operations: [Operation]
+    operations: [Operation, MapOperation]
```

Comment:
Can we keep 1 operation?

Reply:
Sure, I'll just add Map to the Operation